File: fizzbuzz/fizzbuzz.py
def fizzer(i):
  output = ""
  if i % 3 == 0: output += "Fizz"
  if i % 5 == 0: output += "Buzz"
  if output == "": output = str(i)
  print(output) 

def main(v) :
  v += 1 # Added one for ease of use
  for i in range(1, v):
    fizzer(i)

main(15)

File: fizzbuzz/fizzbuzz.ex

defmodule Fizzbuzz do
  def main(i) do
    Enum.each(1..i, &fizzer/1)
  end


  def fizzer(i) when rem(i, 15) == 0 do
     IO.puts("FizzBuzz")
  end
  def fizzer(i) when rem(i, 3) == 0 do
     IO.puts("Fizz")
  end
  def fizzer(i) when rem(i, 5) == 0 do
    IO.puts("Buzz")
  end

  def fizzer(i) do IO.puts(i) end

end
